### ISSUE TITLE
Unable to find dependency 'stslib' for mod 'ReplayTheSpireMod'.

### ISSUE THREAD

**Issue by Nightmarewarith** [AstroPenguin642/Replay-the-Spire#88]
Literally just downloaded this mod and title is the message I get, this is literally the only thing that shows up in the log when I try to load this mod. Doesn't matter if I have other mods enabled or not, as soon as I try enabling this mod  this messag eshows up and literally nothing else

**Comment by Luddeeeeeee** [comment]
I also have this same problem. i have tried everything i can come up with to fix it but with no success

**Comment by imakunee** [comment]
Might sound like a dumb question but you DO have stslib (there's another mod that is a dependency for this mod called stslib) installed and activated correct? 'Cause I'm using replay the spire with loading perfectly fine.

**Comment by Luddeeeeeee** [comment]
Yeah I figured that out. My bad, didn't read the whole download page.
